CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Fix rolling timer being rescheduled rapidly due to leeway (#1106 / #1107)
 - Fix -didArchiveLogFile: returning the file name instead of the file path (#1078)
 - Fix setxattr() function usage
+- Fix NSDateFormatter thread safety
```

Comment:
Btw. please add the pull request ID to the changelog (maybe also for your last PR, since I missed it there 😅). Makes it easier to get to the full discussion (see entries above). 🙂

Reply:
See 860c578